feat: SELinux policy

pvoborni opened the pull request:
Commits:

**feat: selinux policy**

Adds SELinux policy for daemon running as a systemd service.

**build: selinux rpm package**

Adds build of host-metering-selinux package containing the SELinux
policy.

ehelms commented on contrib/rpm/host-metering.spec.in:
```diff
@@ -34,12 +37,24 @@ BuildRequires: golang >= 1.19
 %gopkg
 
+%package selinux
+Summary:	    SELinux policy module for host-metering
+BuildArch:      noarch
```
```suggestion
Summary: SELinux policy module for host-metering
BuildArch: noarch
```

ehelms reviewed (dismissed):
I am not an SELinux expert, this looks like a good start.

pvoborni commented:
Applying the "suggestion" removed approval.  So I fixed it also to "be aligned", added missed build artifacts from `make build-selinux` to `.gitignore` and `make clean` and rebased this PR.
